File: frontend-lsport/src/app/entity/Intermediacao.ts
export class Intermediacao {
  id: number;
  jogadorTransferido: string;
  equipaAntiga: string;
  equipaNova: string;
  anoIntermediacao: number;
  photoPath:string;

  constructor(id: number, jogadorTransferido: string, equipaAntiga: string, equipaNova: string, anoIntermediacao: number, photoPath:string) {
    this.id = id;
    this.jogadorTransferido = jogadorTransferido;
    this.equipaAntiga = equipaAntiga;
    this.equipaNova = equipaNova;
    this.anoIntermediacao = anoIntermediacao;
    this.photoPath = photoPath;
  }
}

File: backend-lfsport/src/main/java/com/ngola/backendlfsport/entity/Intermediacao.java
package com.ngola.backendlfsport.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;

@Entity
@Data
public class Intermediacao {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    private String jogadorTransferido;
    private String equipaAntiga;
    private String equipaNova;
    private long anoIntermediacao;
    private String photoPath;
}
